api/telephone.go

```diff
@@ -10,6 +10,7 @@ var (
 	formatTelephoneDomestic      = regexp.MustCompile("\\d{3}-?\\d{3}-?\\d{4}")
 	formatTelephoneInternational = regexp.MustCompile("\\d{3}-?\\d{4}-?\\d{4}")
 	formatTelephoneDSN           = regexp.MustCompile("\\d{3}-?\\d{4}")
+	formatNumberType             = regexp.MustCompile("(Home|Work|Cell|NA|^$)")
```

Comment:
`NA` is no longer an option since choosing `Home|Work|Cell` is required, but probably doesn't hurt to keep it here.